packages/edit-widgets/src/components/header/index.js

```diff
@@ -56,7 +53,10 @@ function Header( { setListViewToggleElement } ) {
 					{ hasFixedToolbar && isLargeViewport && (
 						<>
 							<div className="selected-block-tools-wrapper">
-								<BlockContextualToolbar isFixed />
+								<BlockToolbar
+									hideDragHandle
+									variant="unstyled"
```

Comment:
Agreed. `hideDragHandle` is the one previously exposed prop on `<BlockToolbar />`. The drag handle is only shown when it's used as a popover. I also think it's better to have it inverted, like `showDragHandle` with a default of `false`. Since it is a publicly available prop, I didn't think I could easily change it though.

Reply:
How is it public now?

Reply:
`hideDragHandle`? I didn't change anything about that prop, and since `<BockToolbar />` was already public, I though `hideDragHandle` could be used?